File: multi-ads-mediator-sdk/src/main/java/com/mdkhaki/multi_ads_mediator_sdk/MultiAdsMediator.java
package com.mdkhaki.multi_ads_mediator_sdk;

import android.app.Application;

import java.util.Arrays;

import ir.tapsell.sdk.Tapsell;

public class MultiAdsMediator {

    private void test(Application application, String appID, Arrays networks) {

        initialize(application, appID, new NetworkCustom(true, true,true));
    }

    public static void initialize(Application application, String appID, NetworkCustom networkCustom) {

        if (networkCustom.tapsellAds) {
            Tapsell.initialize(application, appID);
        }
    }

}

File: multi-ads-mediator-sdk/src/main/java/com/mdkhaki/multi_ads_mediator_sdk/MultiAdsMediator.kt
package com.mdkhaki.multi_ads_mediator_sdk

import android.app.Application
import ir.tapsell.sdk.Tapsell
import java.util.*

class MultiAdsMediator {
    private fun test(application: Application, appID: String, networks: Arrays) {
        initialize(application, appID, NetworkCustom(tapsellAds = true,unityAds = true,chartBoostAds = true))
    }

    companion object {
        fun initialize(application: Application?, appID: String?, networkCustom: NetworkCustom) {
            if (networkCustom.tapsellAds) {
                Tapsell.initialize(application, appID)
            }
        }
    }
}